src/molecule_podman/driver.py

```diff
@@ -184,7 +188,7 @@ def login_options(self, instance_name):
         return {"instance": instance_name}
 
     def ansible_connection_options(self, instance_name):
-        return {"ansible_connection": "podman"}
+        return {"ansible_connection": "podman", "ansible_podman_executable": f"{podman_exec}"}
```

Comment:
from black output:
please format it like
```diff
     def ansible_connection_options(self, instance_name):
-        return {"ansible_connection": "podman", "ansible_podman_executable": f"{podman_exec}"}
+        return {
+            "ansible_connection": "podman",
+            "ansible_podman_executable": f"{podman_exec}",
+        }


```

Reply:
Should be done now :)